### PR TITLE
Fix CMaNGOS non PCH build on High Sierra

### PR DESCRIPTION
**Pull request by Lillecarl**
PCH is still broken because unclear reasons.

### CODE REVIEW

**Comment by killerwife** [comment]
@cala any comments on this? I don't have apple development.

**Comment by Lillecarl** [comment]
I should note that i'm building with gcc

File: src/shared/Database/DatabaseMysql.h [hunk 1]
```diff
@@ -27,6 +27,8 @@
 
 #ifdef _WIN32
 #include <mysql/mysql.h>
+#elif __APPLE__
```
Comment (Laizerox) [inline src/shared/Database/DatabaseMysql.h]: Why don't you just replace #ifdef and #elif with `#if defined(_WIN32) || defined(__APPLE__)` ?

Comment (Lillecarl) [reply src/shared/Database/DatabaseMysql.h]: Why why is because i didn't know you could do that with preprocessor macros 😄

**Comment by Lillecarl** [comment]
I'll have to revisit this, apple ecosystem doesn't seem to be very consistent across CI and actual OS.

**Pull request closed without merging**